WeatherPlannerDemo/Features/CityDetails/UserInterface/ViewControllers/CityDetailsPresenter.swift

```diff
@@ -1,34 +1,32 @@
-import Foundation
+import RxSwift
 
 final class CityDetailsPresenter {
     
     var title: String?
     
     var cityViewModel: CityViewModel
-    var airPollutionDetails: [AirPollutionDetailsViewModel] = []
+    var airPollutionDetails: Observable<[AirPollutionDetailsViewModel]> = .just([])
```

Comment:
You don't need to set the initial value because this property is initialized in init.